### PR TITLE
Test 1 proposal

### PR DESCRIPTION
**Pull request by marianstahl**
FYI @mmikhasenko @DhruvanshuSan

### CODE REVIEW

**Comment by mmikhasenko** [comment]
merged in main, and polished by running `pre-commit run -a`.
A few notes:
 - the functions that are exported (`export rbw`) one can call without the module name (no need for `DataAnalysisWS2425.rbw`)
 - in relation to that, I suggest to rename `gauss` - make it less general
 - `rbw` => `relativistic_breit_wigner`
- also, replaced `==` to `≈` (`\approx`) for floats

Could you please confirm that these test can be satisfied

File: tutorials/tutorial-01.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+We will need some of the p.d.f.s in un-normalized form (such that the optimizations during fitting can work faster)
+
+1. Gaussian
+2. Polynomial
+3. Relativistic Breit-Wigner
+4. Voigt profile
+
+To pass the tests, their signatures should look like this:
+
+1. `gauss(x; μ, σ, a)`, where `a` is an arbitrary normalization factor
```
Comment (mmikhasenko) [inline tutorials/tutorial-01.md]: suggest renaming to make to more specific and clear.
It does not have to be too short

File: tutorials/tutorial-01.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+We will also use the sampling methods and generalize them to take any function as input. Their signature looks like this:
+
+```julia
+function sample_rejection(f, n, support, nbins=1000)
+# f is the function,
+# n the sample size (passing the method),
+# support is a tuple specifying in which range the function will be sampled
+# nbins is the number of equidistant points in "support" for finding the maximum of the function (the default value is 1000)
+```
```
Comment (mmikhasenko) [inline tutorials/tutorial-01.md]: Suggest placing here a docstring that students can reuse

(chatgpt)

```
"""
    sample_rejection(f, n, support; nbins=1000)

Generates `n` samples using the rejection sampling method for a given function `f` over a specified `support` range.

# Arguments
- `f::Function`: The function to sample from.
- `n::Int`: The number of samples to generate.
- `support::Tuple{T, T}`: A tuple specifying the range `(a, b)` where the function `f` will be sampled.
- `nbins::Int`: Optional. The number of equidistant points in `support` to find the maximum of `f`. Default is `1000`.

# Returns
- An array of `n` samples generated from the distribution defined by `f`.

# Example
```julia
data = sample_rejection(cos, 1000, (0, pi); nbins=1000)
```
"""
```

File: tutorials/tutorial-01.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+
+```julia
+function sample_inversion(f, n, support, nbins=1000)
+# Similar to sample_rejection.
+# Here, nbins is the number of bins in which the c.d.f. is pre-computed. See lecture-02-a for details.
+```
```
Comment (mmikhasenko) [inline tutorials/tutorial-01.md]: Replace to a docstring

File: tutorials/tutorial-01.md [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+
+Finally, you will be implementing a method to compute the correlation between two arrays:
+
+```julia
+correlation(A, B)
+```
```
Comment (mmikhasenko) [inline tutorials/tutorial-01.md]: @DhruvanshuSan and students found this function already implemented in `Statistics.jl`

**Review by mmikhasenko — CHANGES_REQUESTED** [review]

**Comment by marianstahl** [comment]
Hi @mmikhasenko . Comments are implemented, please check again

**Review by mmikhasenko — APPROVED** [review]

**Pull request merged**